src/data/activities/media.js

```diff
@@ -12,6 +12,7 @@ import technovationGirls from '../../assets/events/technovation_girls.png';
 import incode2023 from '../../assets/events/incode_2030.jpg';
 import webinarAubay from '../../assets/events/webinar_aubay.jpg';
 import comunicacoesSet2023 from '../../assets/events/comunicacoes_Set_2023.png';
+import digitalWithPurpose2023 from '../../assets/events/digital_with_purpose_2023.jpg'
```

Comment:
[O linter no `build` nas PR checks](https://github.com/As-Raparigas-do-Codigo/ardc-website/actions/runs/6726600401/job/18283141497?pr=149) está a queixar se aqui da falta de `;` no final. Podes acrescentar por favor? O teu código funciona na mesma, isto é só um check que temos para seguirmos o mesmo standard de estilo de código ao longo do projeto.

```suggestion
import digitalWithPurpose2023 from '../../assets/events/digital_with_purpose_2023.jpg';
```

Também porque fiz update do teu branch com `dev`, podes fazer `git pull` no teu repo local?
Podes também aceitar a sugestão neste comentário, se for mais fácil.

Reply:
Obrigada pela correção! Já fiz commmit da sugestão